jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -34,6 +39,7 @@ class LineListTool(PluginTemplateMixin):
     rs_slider_step_auto = Bool(True).tag(sync=True)
     rs_slider_step = Float(0.01).tag(sync=True)
     rs_redshift_step = Float(1).tag(sync=True)
+    # TODO: rs_wavelength_step (although could be across various units and wavelength/freqeuncy...)
```

Comment:
Is there like a follow-up to clarify this TODO?

Reply:
I'll remove this comment (was probably just a missed insertion during rebasing) and cleanup the naming of `rs_redshift_step` -> `rs_rv_step` since that is now how its being applied (as the step size on the RV input element).